src/test/java/lotto/domain/LottoRankTest.java

```diff
@@ -15,32 +15,33 @@ class LottoRankTest {
 
     @ParameterizedTest
     @MethodSource("lottoRankProvider")
-    void of(int count, LottoRank rank) throws Exception {
+    void of(int count, LottoRank rank, boolean matchBonus) throws Exception {
```

Comment:
현재 애플리케이션 로직에서 `checked exception`을 사용하는 곳이 없으니, 
예외를 던지는 부분은 없어도 될 것 같습니다.
```suggestion
    void of(int count, LottoRank rank, boolean matchBonus) {
```

Reply:
영한님으 로드맵을 수강하다가 영한님께서 tdd 라고 템플릿 단축키를 만들어서 사용하시는 것을 보았습니다!

``` java
@Test
void '메서드 이름()' throws Exception {
   //given

   //when
   
   //then
}
```

특별히 체크 예외가 없으셔도 throws Exception 을 사용하시는 것 같아서 저도 러프하게 사용했습니다. 이런 부분도 꼼꼼히 체크하는 것이 좋을까요?? 아니면 러프하게 사용해도 되는 부분일까요??

Reply:
> 특별히 체크 예외가 없으셔도 throws Exception 을 사용하시는 것 같아서 저도 러프하게 사용했습니다. 이런 부분도 꼼꼼히 체크하는 것이 좋을까요?? 아니면 러프하게 사용해도 되는 부분일까요??

개인적으로는 불필요한 코드는 최대한 작성하지 않는 편이 좋다고 생각하는 주의입니다만
어디까지나 지극히 개인적인 부분이기 때문에, 편하신대로 작성해주셔도 무방하다고 생각합니다.